datahub-frontend/play.gradle

```diff
@@ -23,42 +23,56 @@ dependencies {
     play('org.springframework:spring-core:5.2.3.RELEASE')
     play('com.fasterxml.jackson.core:jackson-databind:2.9.10.4')
     play('com.nimbusds:nimbus-jose-jwt:7.9')
-    play('com.typesafe.akka:akka-actor_2.11:2.5.16')
+    play('com.typesafe.akka:akka-actor_2.12:2.5.16')
     play('net.minidev:json-smart:2.4.1')
     play('io.netty:netty-all:4.1.44.Final')
   }
-  play project(":datahub-graphql-core")
-  play project(":metadata-service:auth-api")
-
-  play externalDependency.jettyJaas
-  play externalDependency.graphqlJava
-  play externalDependency.antlr4Runtime
-  play externalDependency.antlr4
-
-  play externalDependency.jerseyCore
-  play externalDependency.jerseyGuava
-
-  play externalDependency.pac4j
-  play externalDependency.playPac4j
-  play externalDependency.shiroCore
-  play externalDependency.playCache
-  play externalDependency.playWs
-  play externalDependency.kafkaClients
-
-  playTest externalDependency.mockito
-  playTest externalDependency.playTest
-
-  playRun externalDependency.lombok
-  playRun externalDependency.guice
-  playRun externalDependency.playDocs
-  playRun externalDependency.playGuice
-  playRun externalDependency.logbackClassic
+  compile project(":datahub-graphql-core")
```

Comment:
awesome. glad to get rid of the whole "play" and "playTest" thing

Reply:
Yep! Had to remove them for the new plugin, no longer worked